playbooks/byo/preflight/Dockerfile

```diff
@@ -8,6 +8,7 @@ MAINTAINER "Customer response and diagnostics team" <dev@lists.openshift.redhat.
 ADD . /opt/app-root/src/
 ENV PLAYBOOK_FILE=playbooks/byo/preflight/check.yml \
     INVENTORY_FILE=hosts \
-    ANSIBLE_HOST_KEY_CHECKING=False
+    ANSIBLE_HOST_KEY_CHECKING=False \
+    ALLOW_ANSIBLE_CONNECTION_LOCAL=false
```

Comment:
Ah, took me some time to realize this is something particular to https://github.com/aweiteka/playbook2image, not Ansible.

You told me yesterday, but I think I didn't fully understand yet why we need that.
I see it [runs sed through the inventory](https://github.com/aweiteka/playbook2image/blob/bd47ea08abd229e5f575290577e5d6925abbb2ce/.s2i/bin/run#L26), but if we still had a reference to "localhost" there, what difference does it make if Ansible connects to localhost (the ephemeral container) via SHH or the local method?